src/Unirota.API/Controllers/SolicitarEntradaController.cs

```diff
@@ -0,0 +1,38 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Unirota.API.Controllers.Common;
+using Unirota.Application.Commands.SolicitarEntrada;
+using Unirota.Application.Services;
+using Unirota.Application.Common.Interfaces;
+
+namespace Unirota.API.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class SolicitacaoEntradaController : BaseApiController
+{
+    private readonly ICurrentUser _currentUser;
+
+    public SolicitacaoEntradaController(IServiceContext serviceContext, ICurrentUser currentUser) : base(serviceContext)
+    {
+        _currentUser = currentUser;
+    }
+
+    [HttpPost]
+    [Authorize]
+    public async Task<IActionResult> SolicitarEntrada([FromBody] SolicitarEntradaGrupoCommand command)
+    {
+        if (command.GrupoId == 0)
```

Comment:
Fazer esse tratamento dentro do handler ou service. Pode ser simplificado da seguinte maneira:
1. Ao usar o repository para fazer a consulta
2. verifica se o grupo é null
3. se for null adiciona o erro ao serviceContext.addError

Reply:
Corrigido